Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -333,9 +349,9 @@ int main(int argc, char** argv)
         frequency_menu.add_action(*action);
     };
 
-    make_frequency_action(1);
-    make_frequency_action(3, true);
-    make_frequency_action(5);
+    make_frequency_action(1, frequency == 1);
+    make_frequency_action(3, frequency == 3);
+    make_frequency_action(5, frequency == 5);
```

Comment:
`make_frequency_action` captures `frequency`, so instead of repeating the frequencies for these checks, can we just do it once in the lambda?

Reply:
Yeah, is it good now?